kolibri/plugins/setup_wizard/templatetags/setup_wizard_tags.py

```diff
@@ -0,0 +1,44 @@
+"""
+Learn template tags
```

Comment:
Looks like this file needs some renaming in the comments/docstrings, in various places.